examples/gatsby/package.json

```diff
@@ -33,13 +33,13 @@
     "@nacelle/shopify-checkout": "^0.0.6",
     "@react-hook/media-query": "^1.1.1",
     "fuse.js": "^6.4.1",
-    "gatsby": "^2.32.13",
+    "gatsby": "^4.5.5",
     "gatsby-alias-imports": "^1.0.6",
-    "gatsby-plugin-emotion": "^4.5.0",
-    "gatsby-plugin-image": "^1.1.2",
-    "gatsby-plugin-sharp": "^2.9.0",
-    "gatsby-source-filesystem": "^2.6.1",
-    "gatsby-transformer-sharp": "^2.7.0",
+    "gatsby-plugin-emotion": "^7.5.0",
```

Comment:
With this upgrade to `gatsby-plugin-emotion@7.x`, I think we need to install some new `@emotion` dependencies because:

> This plugin supports Emotion v11+. Older versions should use versions of this plugin which support Emotion 8 and 9. Check out the Emotion 10 migration guide for more information on how to upgrade.

The problem shows up after `npm install`ing in `examples/gatsby` (followed by `npm run bootstrap` in the monorepo root). When I did that, there were changes to `package-lock.json`, and

```
Module not found: Error: Can't resolve '@emotion/react'
```

in multiple files when running `gatsby develop`.

I think this problem will be resolved by removing the old emotion packages, installing the dependencies listed in [the `gatsby-plugin-emotion` README](https://www.gatsbyjs.com/plugins/gatsby-plugin-emotion), and updating the setup in `examples/gatsby/gatsby-browser.js` to:

```js
import { Global } from '@emotion/react'
```

Reply:
So I'm not getting this issue at all. Though I install all of the example dependencies by running `npm run bootstrap` from the root folder. 

I can make this change if you still have the issue persist though. I'd delete your `examples/gatsby` node_modules & `examples/gatsby` `package-lock.json` just to make sure. I've found that I had a _lot_ of dependency issues with  Lerna and this was the only way I could fix them.

Reply:
I think we should make these changes, since `gastby-plugin-emotion@7.x` is explicitly not compatible with the version of Emotion we're using in `examples/gastby`.

Reply:
Cool. I figured we would want to. Just wasn't sure if we wanted to in this PR or not considering the timing. We probably  need a task in general to get all of our `nacelle-react` packages up to date, even if we're working on moving them to `nacelle-js`.

Reply:
@mdarrik I think you're safe to disregard the README update I suggested before - I missed that [`getNode(nodeId)`](https://www.gatsbyjs.com/docs/reference/config-files/node-api-helpers/#getNode) is [backwards-compatible](https://v2.gatsbyjs.com/docs/reference/config-files/node-api-helpers/#getNode), and I don't see any other red flags for older versions of Gatsby.

But with that in mind, I wonder if:

https://github.com/getnacelle/nacelle-react/blob/5ea1b7e0c3e35f593508b9e607895254728e0d40/packages/gatsby-source-nacelle/package.json#L30

is the right choice?

I guess it won't hurt any of our active customers, who are all either on Gatsby 4 already or in the process of upgrading to Gatsby 4. I'll defer to your judgement about the best combo of that peerDep + README info.